Game/Levels/LessOrEqual/L08le_total.lean

```diff
@@ -33,7 +33,7 @@ Statement le_total (x y : ℕ) : x ≤ y ∨ y ≤ x := by
   use e + 1
   rw [succ_eq_add_one, add_assoc]
   rfl
-  Hint (hidden := true) "Now `cases h2 with e he`."
+  Hint (hidden := true) "Now `cases h2 with e h2`."
```

Comment:
```suggestion
  Hint (hidden := true) "Now `cases {h2} with {e} {h2}`."
```

Variables from context are supposed to be escaped with `{}` in Hints so that their names can be replaced with the player's local name.

Reply:
Would you mind quickly doing that for all hints in this exercise?